pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+# SAMPL6 pKa Challenge Instructions
+
+Challenge timeframe: Oct 24, 2017 to Jan 10, 2018  
+
+This challenge consists of predicting microscopic and macroscopic pKas of 24 small organic molecules. These fragment-like small molecules are selected for their similarity to kinase inhibitors and for experimental tractability. Our aim is to evaluate how well current pKa prediction methods perform with drug fragment-like molecules through blind predictions.
+
+Three formats of pKa prediction results will be evaluated:
+1. microscopic pKa values and related microstates
+2. microstate populations as a function of pH
+3. macroscopic pKa values
+
+The following subsections describe the molecules included in this challenge, the experimental conditions and measurements, the quantities to be predicted, and how prediction results must be submitted.
+
+## Due Date
+
+Your predictions must be uploaded on the D3R SAMPL6 web-page by January 10, 2018. The experimental results will be released immediately after the challenge closes. You must use the provided templates to upload your predictions to the [SAMPL website](https://drugdesigndata.org/about/sampl6). Additional information on using these templates is provided below.
+
+## Computational prediction methods
+
+You may use any method(s) you like to generate your predictions; e.g., molecular mechanics or quantum mechanics based methods, QSPR, empirical pKa prediction tools etc.
+
+## 24 small molecules included in the pKa challenge
+![pKa_challenge_small_molecules](images/pKa_challenge_small_molecules.jpg)
+A list of SAMPL6 pKa challenge small molecules canonical isomeric SMILES and molecule IDs can be found here: [/physical_properties/pKa/molecule_ID_and_SMILES.csv](physical_properties/pKa/molecule_ID_and_SMILES.csv). Counterions, where present in solid formulations, were included in canonical isomeric SMILES for the sake of completeness, although no significal effect is expected from the presence of chloride counterions as experiments were conducted in ionic-strength adjusted medium with KCl.
+
+## Experimental details
+pKa measurements were collected using spectrophotometric pKa measurements with Sirius T3 instrument by Mehtap Isik from the Chodera Lab at MSKCC with the support of Merck Rahway, Preformulation Department. 
```

Comment:
Do you want to list Dorothy, Tim, and Brad, at least in an "Acknowledgments" section?

Reply:
That sounds important.

Reply:
A great point. I added their names.